src/lib/address/address.ts

```diff
@@ -66,6 +67,7 @@ const vanityResolversByChainId: VanityAddressResolversByChainId = {
   [btcChainId]: [resolveUnstoppableDomain],
   [ethChainId]: [resolveEnsDomain, resolveUnstoppableDomain],
   [dogeChainId]: [],
+  [ltcChainId]: [],
   [cosmosChainId]: [],
   [osmosisChainId]: [],
```

Comment:
We don't need to explicitly set any of these to an empty array, as we fallback to one here: https://github.com/shapeshift/web/blob/8b2f2260b55485620462a4c6d6fb35e0cb8c4161/src/lib/address/address.ts#L40

Lines 69 to 72 can be deleted entirely.